spec/features/post_index_page_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index  page', type: :feature do
+  describe 'post index Page' do
+    before(:each) do
+      @user2 = User.create(name: 'David', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                           bio: 'Teacher From Country', posts_counter: 5)
+      visit root_path
+
+      @post1 = Post.create(user: @user2, title: 'My First Post', text: 'This is my first post', comments_counter: 1,
+                           likes_counter: 2)
+      @post2 = Post.create(user: @user2, title: 'My second Post', text: 'This is my first post', comments_counter: 2,
+                           likes_counter: 3)
+      @post3 = Post.create(user: @user2, title: 'My third Post', text: 'This is my first post', comments_counter: 3,
+                           likes_counter: 4)
+      @post4 = Post.create(user: @user2, title: 'My fourth Post', text: 'This is my first post', comments_counter: 4,
+                           likes_counter: 5)
+      visit user_path(@user2.id)
+
+      @comment1 = Comment.create(post: @post1, user: @user2, text: 'Hi Tom!')
+      @comment2 = Comment.create(post: @post2, user: @user2, text: 'Hi Tom!')
+      visit user_posts_path(@user2.id)
+    end
+
+    scenario 'I can see the user\'s profile picture.' do
+      expect(page).to have_css('div.user-icon')
+    end
+
+    scenario 'I can see the user\'s username' do
+      expect(page).to have_content('David')
+    end
+
+    scenario 'I can see the number of posts the user has written' do
+      post = Post.all
+      expect(post.all.count).to eql(42)
```

Comment:
* Kindly fix this failing test 🙏 
![image](https://user-images.githubusercontent.com/52497246/206099116-32355a36-e081-499d-8e92-d0699f513f37.png)